core/rule-engine/rule-engine-components/pom.xml

```diff
@@ -71,5 +71,21 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-mvel</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-quartz</artifactId>
+            <version>${camel.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.cronutils</groupId>
+            <artifactId>cron-utils</artifactId>
+            <version>9.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.simplejavamail</groupId>
+            <artifactId>simple-java-mail</artifactId>
+            <version>8.12.4</version>
```

Comment:
Will the version be defined in the parent pom?

Reply:
Done.